x-pack/plugins/security/public/views/management/roles.js

```diff
@@ -35,7 +35,17 @@ routes.when(ROLES_PATH, {
     $scope.deleteRoles = () => {
       const doDelete = () => {
         $q.all($scope.selectedRoles.map((role) => role.$delete()))
-          .then(() => toastNotifications.addSuccess(`Deleted ${$scope.selectedRoles.length > 1 ? 'roles' : 'role'}`))
+          .then(() => toastNotifications.addSuccess(
+            i18n('xpack.security.views.management.roles.deleteRoleTitle', {
+              values: {
+                valueText: $scope.selectedRoles.length > 1 ?
```

Comment:
use plural from ICU format for such cases. Come to me and I'll show you how to do it